### ISSUE TITLE
[$250] Screen Reader: Workspace-Workflow: The button has interactive elements nested inside

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76909]
## Action Performed:
Prerequisite:
User is logged in
User has created at least one workspace

1. Using JAWS + Chrome, open the page https://new.expensify.com/workspaces
2. Navigate using TAB to any workspace created, press enter to activate
3. Navigate using TAB to 'Workflows' located in the middle navigation column, press enter to activate
4. Navigate using TAB to the 'Add Approvals' button
5. Continue using TAB to navigate again, observe the 'Expenses from' and 'Approver' text is focused and announced

## Expected Result:
When users navigate to the 'Add Approvals' button on the Workflows page, the element will only be focused and announced once, and there will be no additional controls nested within the button element.
-"Add Approvals, button"

## Actual Result:
When users navigate to the 'Add approvals' button, the nested 'Expenses from' and 'Approver' elements create unnecessary tab stops as these nested elements take the user to the same menu
-"Add approvals, button"
-"Expenses from everyone, button"
-"Approver, Test Account, button"

HTML:
<button aria-label="Add approvals" role="button" data-selection-scrapper-hidden-element="true" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-171az5r r-1xfd6ze r-rs99b7 r-nsbfu8 r-18u37iz r-1wtj0ep r-11c0sde r-bvlit7 r-9ffhgg r-lrvibr" type="button" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx">

<div aria-label="Expenses from" role="menuitem" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-17gur6a r-18u37iz r-1wtj0ep r-13qz1uu r-t60dpp r-1mkv55d r-1mdbw0j r-1e084wi r-gy4na3 r-9aemit r-wk8lta" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx">

<div aria-label="Approver" role="menuitem" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-17gur6a r-18u37iz r-1wtj0ep r-13qz1uu r-t60dpp r-1mkv55d r-1mdbw0j r-1e084wi r-gy4na3 r-9aemit r-wk8lta" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx">
## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Area issue was found in:
Workspace Workflow

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Avoid nesting interactive elements into each other, as that can cause various unpredictable keyboard and screen reader issues. Combine all the elements and texts into a single button element, or bring the sub-buttons outside of the buttons if they are intended to trigger different actions.
https://www.w3.org/WAI/ARIA/apg/patterns/button/
https://adrianroselli.com/2016/12/be-wary-of-nesting-roles.html

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
https://github.com/user-attachments/assets/6d6c1ede-1149-460e-87eb-3946a355b728

<img width="1916" height="1003" alt="Image" src="https://github.com/user-attachments/assets/416d3f82-a0d4-46b7-82cb-351ef41cde9b" />

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6996240
**Issue reported by:** NA
**Slack conversation:** NA

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~022009715436533231043</li>
        <li>Upwork Job ID: 2009715436533231043</li>
        <li>Last Price Increase: 2026-02-09</li>
<li>Automatic offers: </li>
<ul>
<li>daledah | Contributor | 110077831</li>
</ul></li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @mallenexpensify</details>

**Comment by mallenexpensify** [comment]
@daledah this job is open to all contributors, inc. C+

**Comment by mountiny** [comment]
Updated

**Comment by mallenexpensify** [comment]
### Payment Summary
Contributor: @daledah due $250 via Upwork. 
Contributor+: @dukenv0307 due $250 via NewDot

@daledah can you please accept the job below? Please reply here and tag me once you have. 
https://www.upwork.com/jobs/~022009715436533231043

You were hired/assigned 10 days before your eligibility date

**Comment by quinthar** [comment]
Approved $250 to @dukenv0307
